src/fides/api/ctl/sql_models.py

```diff
@@ -269,6 +270,23 @@ class System(Base, FidesBase):
     ingress = Column(JSON)
 
 
+class SystemScans(Base):
+    """
+    The SQL model for System Scan instances.
+    """
+
+    __tablename__ = "plus_system_scans"
+
+    id = Column(Integer, primary_key=True)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    error = Column(String, nullable=True)
+    is_classified = Column(BOOLEAN, default=False, nullable=False)
+    result = Column(JSON, nullable=True)
+    status = Column(String, nullable=False)
+    system_count = Column(Integer, autoincrement=False, nullable=True)
+    updated_at = Column(DateTime(timezone=True), server_default=func.now())
+
+
 sql_model_map: Dict = {
```

Comment:
I think it needs to be included in the `sql_model_map` as well?

Reply:
I don't believe so. It looks like this is only used to map fideslang types to SQL models. There are several models defined in this file that don't appear as KVPs in this map.